### PR TITLE
set-viewpoint-rotation update the progress bar

### PR DESCRIPTION
**Pull request by prupani-7**
## Description

PR to fix `runtime/kotlin/issues/2443`

- Updated the progress bar slider to be below the MapView

## How to Test

Run the sample on the sample viewer or the repo.

### CODE REVIEW

File: set-viewpoint-rotation/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -7,48 +7,54 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/initial_rotation_text"
+                android:textColor="@color/textColor"
+                android:textSize="18sp"
+                android:textStyle="bold"
+                app:layout_constraintBottom_toBottomOf="@+id/rotationSeekBar"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintStart_toEndOf="@+id/rotationSeekBar"
+                app:layout_constraintTop_toTopOf="@+id/rotationSeekBar" />
```
Comment (shubham7109) [inline set-viewpoint-rotation/src/main/res/layout/activity_main.xml:55]:
```suggestion
            <com.google.android.material.slider.Slider
                android:id="@+id/rotationSlider"
                android:layout_width="0dp"
                android:layout_height="wrap_content"
                android:layout_marginStart="16dp"
                android:stepSize="1"
                android:valueFrom="0"
                android:valueTo="360"
                app:labelBehavior="gone"
                app:layout_constraintBottom_toBottomOf="parent"
                app:layout_constraintEnd_toStartOf="@+id/rotationValueText"
                app:layout_constraintStart_toStartOf="parent"
                app:layout_constraintTop_toTopOf="parent"
                app:tickVisible="false" />

            <TextView
                android:id="@+id/rotationValueText"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:layout_marginEnd="16dp"
                android:text="@string/initial_rotation_text"
                android:textColor="@color/textColor"
                android:textSize="18sp"
                android:textStyle="bold"
                app:layout_constraintBottom_toBottomOf="parent"
                app:layout_constraintEnd_toEndOf="parent"
                app:layout_constraintTop_toTopOf="parent" />
```

Seems like this is the only sample still using `SeekBar`, let's move to use `com.google.android.material.slider.Slider` for the sample. 

Also, noticed that `20dp` top and bottom padding is large, I've provided a suggestion to match the contraints of the layout to the size of the slider.

**Review by shubham7109 — CHANGES_REQUESTED** [review]
Hey @prupani-7, sample looks good and works fine. 

I have a suggestion to instead use the material slider instead of the Android widget `SeekBar`. I would see two benefits here:
1. The theme for the slider would be consistent with usages of slider in other samples
2. The Kotlin implementation would be much more straight forward: 


```kotlin
// material slider implementation
rotationSlider.addOnChangeListener { _, angle, _ ->
    // set the text to the value
    rotationValueText.text = angle.toInt().toString()
    // rotate map view to the progress angle
    lifecycleScope.launch {
        mapView.setViewpointRotation(angle.toDouble())
    }
}

// previous implementation
rotationSeekBar.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
    override fun onProgressChanged(seekBar: SeekBar, angle: Int, b: Boolean) {
        // set the text to the value
        rotationValueText.text = angle.toString()
        // rotate map view to the progress angle
        lifecycleScope.launch {
            mapView.setViewpointRotation(angle.toDouble())
        }
    }
    override fun onStartTrackingTouch(seekBar: SeekBar?) {}
    override fun onStopTrackingTouch(seekBar: SeekBar?) {}
})
```

**Comment by prupani-7** [comment]
Thanks for the suggestion @shubham7109 
Using material design slider seems like a better choice than the Seekbar. Updated the code suggested.

**Review by shubham7109 — APPROVED** [review]
LGTM 👍

**Pull request merged**